controllers/authors.js

```diff
@@ -0,0 +1,26 @@
+const models = require('../models')
+
+const getAllAuthors = async (request, response) => {
+  const authors = await models.Authors.findAll()
+
+  return response.send(authors)
+}
+
+const getAuthorById = async (request, response) => {
+  const { id } = request.params
+
+  const author = await models.Authors.findOne({
+    where: { id },
+    include: [{
+      model: models.Novels,
+      include: [{ model: models.Genres }]
+    }]
+  })
+
+  return author
+    ? response.send(author)
+    : response.sendStatus(404)
+}
```

Comment:
Clearly you didn't actually run your code and hit these routes because you would have caught the case issue. None of your routes work because of it

![image](https://user-images.githubusercontent.com/5775716/82098303-b314ba00-96d2-11ea-89ee-121044d4570d.png)